packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -71,22 +68,40 @@ const defaultConfig: Partial< SnapWPEnv & SnapWPConfig > = {
 	useCorsProxy: process.env.NODE_ENV === 'development',
 };
 
+/**
+ * Default configuration.
+ */
+const defaultConfig: Partial< SnapWPEnv & SnapWPConfig > = {
+	graphqlEndpoint: DEFAULTS.graphqlEndpoint,
+	uploadsDirectory: DEFAULTS.uploadsDirectory,
+	restUrlPrefix: DEFAULTS.restUrlPrefix,
+	corsProxyPrefix: DEFAULTS.corsProxyPrefix,
+	useCorsProxy: DEFAULTS.useCorsProxy,
+};
```

Comment:
How is this (and the duplicate `DEFAULTS` ) either helpful or necessary?

Seems like we're just trying to appease a linter, which is always a bad sign.